### DISCUSSION TITLE
Expand fields of models / Frontend api

### DISCUSSION THREAD

**Discussion by souksou** [swellstores/community#335] [Help] [answered]
Hello guys,

I just have a question regarding the exposition of some custom **collection** field.

Actually, the products API returning me:

```
products
--- collections
----- id
```

I want to add name in my collections array like this:

```
products
--- collections
----- id
----- name
```

I tried to add permissions directly in the swell console:
`

> "expands": {
>   "collections_detail": {
>           "url": "/collections",
>           "params": {
>             "id": "collections.id"
>           },
>           "data": {
>             "expand": [
>               "parent.children"
>             ]
>           }
>         }
>   }

`

Do you know how can i aggregate/expands two models by array id ? 

Thanks a lot for your help

**Comment by sags95** [comment, marked as answer]
Hey @souksou, modifying the permissions is more for allowing you update certain fields from the frontend api, or expose certain fields to the frontend api. In this case for a custom field on the /Products model, you should be able to specify this field with the **content** namespace when defining it within the Models screen. You'll see this in the "Advanced options" section when creating the field in the Developer > Models screen.
